shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -440,6 +443,7 @@ - (BOOL)createShell:(NSString*)entrypoint libraryURI:(NSString*)libraryURI {
                    << entrypoint.UTF8String;
   } else {
     [self setupChannels];
+    [[self lifecycleChannel] sendMessage:@"AppLifecycleState.detached"];
```

Comment:
A bit strange that we're imperatively triggering an edge-triggered signal to set a level-triggered signal. Just letting window._initialLifecycleState = detached seems semantically more correct but also a bit riskier.